MagazineLayout.xcodeproj/project.pbxproj

```diff
@@ -162,10 +162,11 @@
 		93A1C01521ACED0100DED67D /* Types */ = {
 			isa = PBXGroup;
 			children = (
-				93A1C01821ACED0100DED67D /* MagazineLayoutItemSizeMode.swift */,
-				93A1C01A21ACED0100DED67D /* MagazineLayoutHeaderVisibilityMode.swift */,
-				93A1C01621ACED0100DED67D /* MagazineLayoutBackgroundVisibilityMode.swift */,
+				93A1C02021ACED0100DED67D /* MagazineLayout+Default.swift */,
 				93A1C01721ACED0100DED67D /* MagazineLayout+SupplementaryViewKinds.swift */,
+				93A1C01621ACED0100DED67D /* MagazineLayoutBackgroundVisibilityMode.swift */,
+				93A1C01A21ACED0100DED67D /* MagazineLayoutHeaderVisibilityMode.swift */,
+				93A1C01821ACED0100DED67D /* MagazineLayoutItemSizeMode.swift */,
```

Comment:
Was the ordering change to these files intentional? I'd prefer to keep the order in the Xcode navigator as follows:
```
MagazineLayout+Default.swift
MagazineLayoutItemSizeMode.swift
MagazineLayoutHeaderVisibilityMode.swift
MagazineLayoutBackgroundVisibilityMode.swift
MagazineLayout+SupplementaryViewKinds.swift
```

Reply:
(Just put the new `MagazineLayout+Default.swift` at the bottom, since it's probably the least commonly used)

Reply:
I just sorted by name. No problem, will revert